docs/examples/Power Flow Example.ipynb

```diff
@@ -38,14 +38,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
```

Comment:
please restart kernel, clear output and rerun again to set the execution count back to `1`